feature/TC-03 Styled System and Styled Component

hqadeer-nisum-com opened the pull request:
- Create an example pages using with style component.
- Create an style system, create example theme which is using in the components.

Commit:
feature/TC-01 Styled System and Styled Component

- Create an example pages using with style component.
- Create an style system, create example theme which is using in the components.

hiqbalmallick commented on package.json:
```diff
@@ -8,11 +8,14 @@
     "@types/jest": "^27.5.2",
     "@types/node": "^16.11.48",
-    "@types/react": "^18.0.17",
-    "@types/react-dom": "^18.0.6",
```
why these two dev dependencies are deleted?

hqadeer-nisum-com replied:
because its style component so thats why i delete them and add them into routing pr.

hiqbalmallick replied:
they come with the default installation of react app using typescript, plus the code is moved below with a separate dev-dependencies block if you can see.

hqadeer-nisum-com replied:
oh yeah yeah hamza bhai i saw that in my code these dependencies added in my package.json file. but these dependencies added twisty by mistake so thats why i did remove that, do i add them again ?

hiqbalmallick replied:
move them up here again from below.

hqadeer-nisum-com replied:
resolved hamza bhai.

hiqbalmallick commented on src/example/pages/About.tsx:
```diff
@@ -0,0 +1,19 @@
+  return (
+    <div>
+      <p>about page</p>
+      <div>
+        <div>
+          <Button>Back To Home</Button>
+        </div>
+      </div>
+    </div>
```
div inside empty div?

hqadeer-nisum-com replied:
i will change it shortly.

Commit:
feature/TC-01 style component

- comment resolved

Commit:
Merge branch 'develop' into feature/TC-01

Commit:
feature/TC-01

- set all structure of theme and global style
- create separate files of typography, colors and shapes
- delete extra components which had use for testing purpose

Commit:
feature/TC-01

- conflicts resolved

hiqbalmallick commented on src/App.tsx:
```diff
@@ -2,15 +2,19 @@ import "./App.css";
-        <Example />
-      </div>
+      <ThemeProvider theme={theme}>
+        <GlobalStyle />
```
Give this a meaningful name, like main layout or something, plus does <Example /> component should also be a part of global theme right?

hqadeer-nisum-com replied:
yes hamza bhai, its a part of our components.

Commit:
feature/TC-01

- comments resolved
